model/host/host.go

```diff
@@ -760,6 +760,20 @@ func FindAllContainers() ([]Host, error) {
 	return hosts, nil
 }
 
+// FindAllRunningParents finds all running hosts that have child containers
+func FindAllRunningParents() ([]Host, error) {
+	query := db.Query(bson.M{
+		StatusKey:        evergreen.HostRunning,
```

Comment:
to brian's earlier comment, this should be 

`StatusKey: bson.M{"$in": evergreen.UphostStatus}`

Reply:
I don't think so, since we only want to put containers on hosts that are truly running. `evergreen.UphostStatus` seems... wrong in general?